src/main/resources/feedbackQuestionConstSumSubmissionFormOptionFragment.html

```diff
@@ -5,6 +5,6 @@
         id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}-${optionIdx}"
         name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
         value="${constSumOptionPoint}" onChange="updateConstSumMessageQn('${questionIndex}')"
-        min="0" step="1">
+        min="${constSumPointsMin}" max="${constSumPointsMax}" step="1">
```

Comment:
For me, Chrome will check the validity when I try to submit the form, which is not so user friendly. Apparently, I think u will need write JS code to check the max/min value.